SingularityUI/app/views/task.coffee

```diff
@@ -61,4 +66,30 @@ class TaskView extends View
             setTimeout (=> @trigger 'refreshrequest'), 1000
 
 
+    executeCommand: (event) ->
+        event.preventDefault()
+        cmd = $("#cmd option:selected").text()
+        options = $('#cmd-option').val()
+        return if @$('button[type="submit"]').attr 'disabled'
+        return if !cmd
+        taskModel = new Task id: @taskId
+        taskModel.runShellCommand(cmd, options)
+        $('#cmd-confirm').text('Command Sent')
+
+
+    cmdSelected: (event) ->
+        cmd = config.shellCommands.filter((cmd) ->
+          return cmd.name == $("#cmd option:selected").text()
```

Comment:
personally I prefer using `===, &&, ||`, but for consistency of how things have been coded in Singularity we should stick with using `is` for `===`, `or` for `||`, `and` for `&&`, etc